src/containers/PipelineRun/PipelineRun.js

```diff
@@ -328,6 +336,13 @@ export /* istanbul ignore next */ function PipelineRunContainer() {
           return isPending(reason, status);
         }
       },
+      {
+        actionText: intl.formatMessage({
+          id: 'dashboard.editAndRun.actionText',
+          defaultMessage: 'Edit and Run'
```

Comment:
nit: `Edit and run`
We use sentence case for UI elements: https://carbondesignsystem.com/guidelines/content/writing-style#use-sentence-case-capitalization